overlore/llm/natural_language_formatter.py

```diff
@@ -81,9 +81,9 @@ def _npc_to_nl(self, npc: NpcEntity) -> str:
 
         characteristics = npc["characteristics"]
 
-        age: int = cast(int, characteristics.age)
-        role: str = cast(str, ROLES[characteristics.role])
-        sex: str = cast(str, SEX[characteristics.sex])
+        age: int = cast(int, characteristics["age"])
```

Comment:
Is this because the class of characteristics isn't a `BaseModel` anymore but a `TypedDict`?

Reply:
Correct. I think we can remove the type casting as well actually. Will test it out.

Reply:
Did you have a chance to test this out?

Reply:
Yep, typecasting was removed in the previous commit